### PR TITLE
[WIP] use mariadb image from IBS

### PR DESCRIPTION
**Pull request by JanZerebecki**

### CODE REVIEW

**Comment by Itxaka** [comment]
Looks like overriding works:
```
Containers:
  mariadb:
    Container ID:  cri-o://8520e01a20a95b6e3c761447a07babb66afa78777697b0d3e935f3e23b1b1fd3
    Image:         registry.suse.de/devel/cloud/10/containers/containers/soc/mariadb:10.2.25
    Image ID:      registry.suse.de/devel/cloud/10/containers/containers/soc/mariadb@sha256:a7dc950e45320781396b5ca5a0291e5a612ae11089be3e5301833add3733f1eb
    Ports:         3306/TCP, 4567/TCP
    Host Ports:    0/TCP, 0/TCP
    Command:
      /tmp/start.py
    State:          Waiting
      Reason:       CrashLoopBackOff
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
```

Unfortunately the pod failed sooo...

`standard_init_linux.go:207: exec user process caused "no such file or directory"`

File: site/soc/software/config/versions.yaml [hunk 1]
```diff
@@ -179,6 +179,7 @@ data:
       libvirt:
         libvirt: "{{ suse_osh_registry_location }}/openstackhelm/libvirt:{{ suse_infra_image_version }}"
       mariadb:
+        mariadb: "registry.suse.de/devel/cloud/10/containers/containers/soc/mariadb:10.2.25"
```
Comment (toabctl) [inline site/soc/software/config/versions.yaml]: out of scope for this PR - but are there plans to make the containers available on OBS? I think we want to also use theses containers for upstream CI.

Comment (JanZerebecki) [reply site/soc/software/config/versions.yaml]: Yes (in SOC-10393).

**Comment by jsuchome** [comment]
Please adapt also the image location for OSH: `playbooks/roles/deploy-osh/templates/mariadb.yaml.j2`

**Pull request closed without merging**